prismic-model/src/events.ts

```diff
@@ -32,6 +32,9 @@ const events: CustomType = {
         isFullyBooked: booleanDeprecated('Fully booked'),
       }),
       body,
+      locations: list('Locations', {
```

Comment:
Do we want to put this next to `place`, given the two fields are similar? And maybe a comment about why we have both?